### PR TITLE
Release notes - 21.10

### PR DESCRIPTION
**Pull request by twood02**
Release notes from 2021

### CODE REVIEW

File: docs/Releases.md [hunk 1]
```diff
@@ -13,6 +13,49 @@ use a date based versioning system.  Now, a release version can look
...
+- [[254](https://github.com/sdnfv/openNetVM/pull/254)] Adds Fair Queue NF that demonstrates how to use advanced rings mode to directly access packets and distribute them to a set of child NFs. Packets are "classified" using a CRC32 hash and assigned to a queue. Queues are then read from in Round Robin order to process packets of different types in a fair way. Contributed by  ([Rohit M P](https://github.com/rohit-mp)) from NITK. 
+- [[277](https://github.com/sdnfv/openNetVM/pull/277)] Adds support for Jumbo frame packets. Enable by adding a `-j` flag to the manager's go.sh script.
+
+
+**Testing and Development Improvements**
+
+- [[296](https://github.com/sdnfv/openNetVM/pull/296)] Adds unit test for NF messaging infrastructure and fixes memory leak related to overflow of message pools [[Issue 293](https://github.com/sdnfv/openNetVM/pull/293)]. 
+- [[297](https://github.com/sdnfv/openNetVM/pull/297)] Adds VS Code profile to simplify debugging of NFs. 
+- [[302](https://github.com/sdnfv/openNetVM/pull/302)] Adds NF chain performance test to measure and plot inter-NF throughput and latency.
+
```
Comment (catherinemeadows) [inline docs/Releases.md:32]:
```suggestion
- [[308](https://github.com/sdnfv/openNetVM/pull/308)] Adds socket ID information to NF and manager logging print statements. 
```

**Review by catherinemeadows — CHANGES_REQUESTED** [review]

**Comment by twood02** [comment]
This should be ready to merge @catherinemeadows . Then we can merge from develop to master

**Comment by catherinemeadows** [comment]
@twood02 Merging is blocked still. Can you change your PR to develop?

**Comment by twood02** [comment]
> @twood02 Merging is blocked still. Can you change your PR to develop?

oof rookie mistake.  should be fixed now.

**Review by catherinemeadows — APPROVED** [review]

**Review by dennisafa — APPROVED** [review]

**Pull request merged**